### PR TITLE
feat: enable Opcache in CLI

### LINKED ISSUE #409: Enabling opcache extension by default in CLI
When running Psalm, it says

```
Install the opcache extension to make use of JIT on PHP 8.0+ for a 20%+ performance boost!
```

which makes sense, some heavy duty data shoveling would probably benefit from Opcache being installed and enabled in CLI.

WDYT?

Note: even if I enable it manually, it doesn't work

```
/project $ php -d zend_extension=opcache.so -d opcache.enable_cli=1 /tools/psalm | head 

Install the opcache extension to make use of JIT on PHP 8.0+ for a 20%+ performance boost!

Target PHP version: 8.2 (inferred from current PHP version).
Scanning files...
Analyzing files...
```

That might be related to Psalm's use of https://github.com/composer/xdebug-handler

### PR DESCRIPTION
**Pull request by dkarlovi**
Closes #409.

### CODE REVIEW

**Comment by jakzal** [comment]
@ciaranmcnulty does it look ok? :)

Summoning Ciaran

**Comment by dkarlovi** [comment]
Wait, I'll add a check for it, I've just learned how to. :smile:

File: Dockerfile [hunk 1]
```diff
@@ -151,8 +151,15 @@ phar.readonly=0
...
 
+COPY --link <<EOF /usr/local/etc/php/conf.d/opcache.ini
+zend_extension=opcache.so
+opcache.enable=1
+opcache.enable_cli=1
+EOF
+
 # Validate the PHP configuration
 RUN php --ini | grep phpqa.ini
+RUN php --rz "Zend OPcache"
```
Comment (dkarlovi) [inline Dockerfile:162]: Note `--rz` is case sensitive, it must be written exactly like this.

**Comment by jakzal** [comment]
Thank you @dkarlovi 🍻

**Pull request merged**